src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -19,14 +19,9 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var host = BuildWebHost(args);
+            InitializeProcess();
 
-            var environment = host.Services.GetService<IEnvironment>();
-            if (environment.IsLinuxContainerEnvironment())
-            {
-                // Linux containers always start out in placeholder mode
-                environment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
```

Comment:
This logic needs to run BEFORE any other host services are initialized. We have code [here](https://github.com/Azure/azure-functions-host/blob/54619304d601ce3df723d06e99f0dff35f2b919c/src/WebJobs.Script.WebHost/Configuration/ScriptApplicationHostOptionsSetup.cs#L46) that checks for placeholder mode to override ScriptPath and other directories to a temp standby folder, which is where the StandbyManager writes the WarmUp function.

In local debugging, I can see that this code was running BEFORE the env var was set.